Sources/FDB/FDB/FDB+Errors.swift

```diff
@@ -279,6 +279,18 @@ public extension FDB {
             return result
         }
 
+        /// Indicates if current error is a native FoundationDB error
+        /// (i.e. errno is less than `8000`, after which custom FDBSwift errors start)
+        public var isNative: Bool {
```

Comment:
`isNative` feels like the wrong term to use - maybe something along the lines of `isFoundationDBError` or `isUnderlyingError` would be better?

Reply:
Nah, I think it's fine. Furthermore, I'll make it internal because no one really need it. In future version I will get rid of it anyway.

Reply:
That's even better!